pyecharts/echarts/option.py

```diff
@@ -1174,12 +1172,7 @@ def zaxis3D(
     return _zaxis3D
 
 
-@collectfuncs
```

Comment:
in the future, I hope these wrapper functions will disappear step by step like this.